api/util/errorLog-function.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const ErrorLog = require('../models/ErrorLog');
+
+async function infoErrorLog(req){
+  let errorSaved = true;
+  const newError = new ErrorLog({
+    userEmail: req.body.userEmail,
+    errorTime: req.body.errorTime,
+    apiEndpoint: req.body.apiEndpoint,
+    errorDescription: req.body.errorDescription
+  });
+  await newError.save()
+    .catch(_ => {
+      errorSaved = false;
+    });
+  return errorSaved;
+}
```

Comment:
add a line between end of function and export line
```suggestion
}

```